docs/install-1.7.md

```diff
@@ -156,4 +157,5 @@ helm install ../../charts/catalog \
         --set apiserver.tls.ca=$(base64 ${SC_SERVING_CA}) \
         --set apiserver.tls.cert=$(base64 ${SC_SERVING_CERT}) \
         --set apiserver.tls.key=$(base64 ${SC_SERVING_KEY})
```

Comment:
Need to add a trailing `\` on this line.

Reply:
done